runner.py

```diff
@@ -70,9 +70,9 @@ def run(pytest, output_file, repeatUntilFailure, testDir, test_slice):
     errorTestPat = re.compile('____ (ERROR.+) ____')
     while True:
         for i, tests in enumerate(test_list_sliced):
-            # testRep = '{}.rep'.format(test.split("/")[-1])
+            testResults = '--junitxml={}-test-results.xml'.format(tests.split("/")[-1])
```

Comment:
These changes are already in `master`.  Not sure why they are showing up here.  Same with all of the GHA related files.